typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "canvas"
+version = "0.0.1"
+repository = "https://github.com/johannes-wolf/typst-canvas"
+entrypoint = "lib.typ"
+authors = [
+  "Johannes Wolf <https://github.com/johannes-wolf>",
+  "fenjalien <https://github.com/fenjalien>"
+]
+license = "Apache-2.0"
+description = "A library for drawing with typst with an API inspired by Tikz and processing."
```

Comment:
Make sure to capitalise **T**ypst and Processing. 

This description should ideally be the same as the one at the top of the readme and in the description of the repo. I do like this one better though.